src/components/CioPlpGrid/CioPlpGrid.tsx

```diff
@@ -32,12 +34,23 @@ export type CioPlpGridProps = {
    * No configurations available yet.
    */
   filterConfigs?: Omit<UseFilterProps, 'facets'>;
+  /**
+   * Used to set the `initialNumOptions` to limit the number of options shown initially.
+   */
+  groupsConfigs?: Omit<UseGroupProps, 'groups'>;
 };
 export type CioPlpGridWithRenderProps = IncludeRenderProps<CioPlpGridProps, UseSearchResultsReturn>;
 
 export default function CioPlpGrid(props: CioPlpGridWithRenderProps) {
-  const { spinner, initialResponse, filterConfigs, sortConfigs, paginationConfigs, children } =
-    props;
+  const {
+    spinner,
+    initialResponse,
+    filterConfigs,
+    sortConfigs,
+    paginationConfigs,
+    groupsConfigs,
```

Comment:
Why a separate config for groupFilters?

Reply:
Mainly because it's a different top-level component